sdk/webpubsub/azure-messaging-webpubsubservice/swagger/README.md

```diff
@@ -67,6 +67,10 @@ directive:
     where: $["paths"]["/api/hubs/{hub}/:generateToken"].post.parameters
     transform: >
         $[2]["x-ms-client-name"] = "roles"
+  - from: swagger-document
+    where: $["paths"]["/api/hubs/{hub}/:generateToken"].post.parameters
+    transform: >
+        $[5]["x-ms-client-name"] = "groups"
```

Comment:
If possible, it is better to re-enable the test https://github.com/Azure/azure-sdk-for-python/tree/1bf8aaa72dbc358f2fb9125d045a32d87499e5c0/sdk/webpubsub/azure-messaging-webpubsubservice/tests. These test could be activated after migration to test proxy: https://github.com/Azure/azure-sdk-for-python/blob/1bf8aaa72dbc358f2fb9125d045a32d87499e5c0/sdk/webpubsub/azure-messaging-webpubsubservice/tests/STATE_OF_TESTS.md

Reply:
let's do this in another pr